package.json

```diff
@@ -9,7 +9,7 @@
         "start": "d2-app-scripts start",
         "format": "yarn d2-style apply",
         "format:staged": "yarn d2-style apply --staged",
-        "lint": "yarn d2-style check",
+        "lint": "yarn d2-style check --no-inline-config",
```

Comment:
I'm not sure why you've added this? I think in general we would want to assume that any disabling of rules via comments in our code is intentional and hence not pass this to the CLI.